hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1336,9 +1336,11 @@
 
             When the chip resets due to alert escalating to cause the chip to reset, verify the
             reset cause to verify the alert crashdump.
+
+            Xref'ed with chip_sw_rstmgr_alert_info.
             '''
       stage: V2
-      tests: []
+      tests: ["chip_sw_rstmgr_alert_info"]
```

Comment:
i actually double checked this with @engdoreis some time back.
I think they felt that there were enhancements needed. I'm not sure if they were planning to write a new test, or enhance the existing one.

Reply:
Oh.. I updated it here(https://github.com/lowRISC/opentitan/pull/14894)
still pending but I added local alert after reviewing with @cindychip

Reply:
Ah great, thanks! Looks like that just updates the `chip_sw_rstmgr_alert_info` test, right?

Should we close this PR here, or just wait until your update is merged?

Reply:
@msfschaffner  Let's keep this open until #14894  is merged 
if there is no urgency.

Reply:
@msfschaffner #14894 is merge. You can close this whenever you can.